examples/preview-panel-in-modal/app.tsx

```diff
@@ -0,0 +1,135 @@
+/** @jsx h */
+import { autocomplete, getAlgoliaResults } from '@algolia/autocomplete-js';
+import algoliasearch from 'algoliasearch/lite';
+import { h, render } from 'preact';
+
+import '@algolia/autocomplete-theme-classic';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+autocomplete({
+  container: '#autocomplete',
+  detachedMediaQuery: '',
+  defaultActiveItemId: 0,
+  debug: true,
```

Comment:
Do we need this?

Reply:
No I forgot about it. Fixed in 750f1d88.